testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2021-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# We also test 'Bootstrapping using the command line' in this feature with the following script:
+# https://github.com/uyuni-project/uyuni/blob/master/java/conf/cobbler/snippets/minion_script
+
+@skip_if_github_validation
+@containerized_server
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler and containerized proxy
+  In order to automate client system installations with a containerized proxy
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
```

Comment:
Why we need this step?
`    And I am on the Systems overview page of this "proxy"
`

Reply:
Probably not indeed.

Let me check also the "traditional" version, the error might come from there by copy-paste.

Reply:
Removed, was indeed a leftover from copy-paste.